Sample Data/Custom/beSECURE_ScanEvents_CL.json

```diff
@@ -0,0 +1,4 @@
+[

```

Comment:
This sample data is very lean - please cover examples of different types of scan status / other fields if applies for a richer sample data set. It enables community to build on top of your data connector  by having rich sample data with different combinations of types.